src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +123,26 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        None, help="Execute immediate all files from the path."
```

Comment:
Do we want to hint that patterns like `@stage/*.sql` work? After reading only help messages I thought it was not a case

Reply:
Good idea

Reply:
We should also have integrations tests for such cases, in fact it would be good to have a template of project where we can test this

Reply:
I think unit test is enough, filter logic is on our side and base on it we execute files on stage. Unit test should cover all or almost all cases https://github.com/snowflakedb/snowflake-cli/pull/949/files#diff-cee995a234660b328c01de5358fc05ea47cd49d77aee3df528787cac5b292e99R484 . If you see any more cases let me know.